site2/docs/helm-deploy.md

```diff
@@ -284,15 +287,15 @@ To enable authorization, you can include this option in the `helm install` comma
 --set auth.authorization.enabled=true
 ```
 
-### CPU and RAM resource requirements
+#### CPU and RAM resource requirements
 
 By default, the resource requests and the number of replicas for the Pulsar components in the Pulsar Helm chart are adequate for small production deployment. If you deploy a non-production instance, you can reduce the defaults to fit into a smaller cluster.
 
 Once you have all of your configuration options collected, you can install dependent charts before installing the Pulsar Helm chart.
 
 ## Install dependent charts
 
-### Install local storage provisioner
+#### Install local storage provisioner
```

Comment:
I think this is not very accurate cause `pulsar-helm-chart` needs a storage provisioner but not must be a local storage provisioner. Users just need to create [storage class](https://kubernetes.io/docs/concepts/storage/storage-classes/) for their k8s cluster and config the storage class name in https://github.com/apache/pulsar-helm-chart/blob/master/charts/pulsar/values.yaml#L361

Reply:
@zymap can you pls review and confirm this point?

Reply:
Is `create storage class and config the storage class` an alternative way to install the storage provisioned compared to the [recommended cmd](https://pulsar.apache.org/docs/next/helm-deploy#install-local-storage-provisioner) in the doc, or a subsequent step?

Reply:
> It also has a counterpart in the [Persistence](https://pulsar.apache.org/docs/next/helm-deploy#persistence) section and even a parameter named local_storage. Should this part be updated as well?

Yeah, I think this part should be updated also. `local_storage` config is for the `streamnative/local-storage-provisioner`. 

> Is create storage class and config the storage class an alternative way to install the storage provisioned compared to the [recommended cmd](https://pulsar.apache.org/docs/next/helm-deploy#install-local-storage-provisioner) in the doc, or a subsequent step?

Creating a storage class and config the storage class is a more general way for storage provisioned. Actually, `helm install pulsar-storage-provisioner streamnative/local-storage-provisioner` is also to install `local-storage-provisioner` in kubernetes as a storage class.

Reply:
@ericsyh sorry for my late response. After a revisit, I understand this `Install local storage provisioner` section as an optional task to perform only if users want to use the local persistent volumes as the persistent storage. Otherwise, they don't need to install any local storage provisioner. Does it make sense to understand this way?

Reply:
Somehow is correct but I don't think it's an optional task because there is no binding relationship between using local persistent volumes and installing `streamnative/local-storage-provisioner`. 

If I want to deploy `pulsar-helm-chart` on k8s with local persistent volumes, there are some provisioners to choose from:
* https://github.com/rancher/local-path-provisioner
* https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner, actually it is the upstream of ` streamnative/local-storage-provisioner`

Reply:
Thanks for the clarification. The section has been updated. PTAL.